tck/src/main/java/org/eclipse/microprofile/openapi/tck/AirlinesAppTest.java

```diff
@@ -702,13 +704,13 @@ public void testEncodingResponses(String type) {
         String s =
                 "paths.'/user/username/{username}'.put.responses.'200'.content.'application/json'.encoding.password.";
         vr.body(s + "contentType", equalTo("text/plain"));
-        vr.body(s + "style", equalTo("form"));
+        vr.body(s, hasOptionalEntry("style", "form"));
```

Comment:
I'm surprised this works when `s` ends with a `.`

Reply:
Yes, I did some additional debugging on this and indeed the correct object is passed into the matcher.